cmd/package_managers.go

```diff
@@ -0,0 +1,157 @@
+// Copyright 2020 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cmd implements Scorecard commandline.
+package cmd
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"time"
+
+	sce "github.com/ossf/scorecard/v4/errors"
+)
+
+var errInvalidInput = errors.New("only one package name must be non-empty")
+
+// TODO(cmd): Add unit tests for this.
+func fetchGitRepositoryFromPackageManagers(npm, pypi, rubygems string) (bool, string, error) {
```

Comment:
IMO tuple is good to read in returns, anything more than `2` should be a struct. It makes it easier to maintain and read. This is with respect to  `bool, string, error`

Reply:
Done.